<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAnyadirUsuarios</class>
 <widget class="QDialog" name="DAnyadirUsuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lCorreo_Modificar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Introduce tu correo electrónico: </string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="gvFotoAnyadir">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>601</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leCorreo_Anyadir">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lAnyadirUsuario">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>AÑADIR USUARIO</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="leApellidos_Anyadir">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lApellidos_Modificar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Introduce tu apellido: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNombre_Anyadir">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lNombre_Modificar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Introduce tu nombre: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leLocalidad_Anyadir">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>370</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lLocalidad_Modificar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Introduce tu localidad: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAceptar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>470</y>
     <width>96</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>470</y>
     <width>96</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lContrasena_Anyadir">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Introduce tu contraseña: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leContrasena_Anyadir">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
